src/main/java/homework_6/map_problems_generator/MapProblemsCollisionGenerator.java

```diff
@@ -0,0 +1,34 @@
+package homework_6.map_problems_generator;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+public class MapProblemsCollisionGenerator {
```

Comment:
I don't see collisions implementation and example

Reply:
I've made examples in the Main class. Should I make a test class instead?

Reply:
Oh... I got it. Sorry, I'll fix it right now!